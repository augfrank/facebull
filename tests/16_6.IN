M0   C0   C1  2540
M1   C0   C2  2012
M2   C0   C7  2614
M3   C0   C8  2578
M4   C0   C9  2458
M5   C0   C10  2798
M6   C0   C11  2131
M7   C0   C13  2032
M8   C0   C14  2532
M9   C0   C15  2531
M10   C1   C0  2934
M11   C1   C2  2930
M12   C1   C3  2536
M13   C1   C4  2529
M14   C1   C5  2992
M15   C1   C7  2322
M16   C1   C9  2097
M17   C1   C11  2359
M18   C1   C14  2765
M19   C2   C0  2400
M20   C2   C1  2880
M21   C2   C3  2266
M22   C2   C4  2898
M23   C2   C7  2708
M24   C2   C8  2091
M25   C2   C9  2265
M26   C2   C10  2616
M27   C2   C12  2436
M28   C2   C15  2915
M29   C3   C1  2867
M30   C3   C4  2146
M31   C3   C5  2849
M32   C3   C6  2286
M33   C3   C7  2964
M34   C3   C8  2166
M35   C3   C9  2382
M36   C3   C11  2052
M37   C3   C13  2023
M38   C3   C14  2043
M39   C3   C15  2271
M40   C4   C0  2787
M41   C4   C1  2717
M42   C4   C3  2906
M43   C4   C6  2305
M44   C4   C7  2580
M45   C4   C8  2017
M46   C4   C11  2654
M47   C4   C12  2337
M48   C4   C13  2111
M49   C4   C14  2266
M50   C5   C0  2216
M51   C5   C1  2633
M52   C5   C2  2360
M53   C5   C3  2805
M54   C5   C4  2868
M55   C5   C6  2233
M56   C5   C7  2852
M57   C5   C8  2211
M58   C5   C9  2221
M59   C5   C10  2355
M60   C5   C11  2182
M61   C5   C13  2701
M62   C5   C14  2264
M63   C5   C15  2066
M64   C6   C0  2456
M65   C6   C1  2971
M66   C6   C2  2300
M67   C6   C3  2994
M68   C6   C4  2699
M69   C6   C7  2477
M70   C6   C9  2918
M71   C6   C10  2532
M72   C6   C11  2776
M73   C6   C14  2247
M74   C6   C15  2467
M75   C7   C0  2140
M76   C7   C1  2767
M77   C7   C2  2984
M78   C7   C3  2134
M79   C7   C4  2830
M80   C7   C5  2323
M81   C7   C6  2468
M82   C7   C11  2073
M83   C7   C12  2909
M84   C7   C13  2608
M85   C7   C14  2504
M86   C8   C0  2071
M87   C8   C1  2934
M88   C8   C3  2835
M89   C8   C4  2139
M90   C8   C5  2404
M91   C8   C7  2477
M92   C8   C10  2387
M93   C8   C12  2123
M94   C8   C13  2270
M95   C8   C14  2069
M96   C9   C0  2639
M97   C9   C2  2752
M98   C9   C3  2820
M99   C9   C5  2280
M100   C9   C6  2285
M101   C9   C7  2567
M102   C9   C8  2342
M103   C9   C10  2670
M104   C9   C11  2280
M105   C9   C12  2655
M106   C9   C13  2549
M107   C9   C14  2132
M108   C9   C15  2604
M109   C10   C0  2493
M110   C10   C1  2661
M111   C10   C2  2920
M112   C10   C8  2714
M113   C10   C9  2984
M114   C10   C13  2067
M115   C10   C15  2845
M116   C11   C0  2620
M117   C11   C1  2864
M118   C11   C3  2733
M119   C11   C4  2057
M120   C11   C5  2205
M121   C11   C6  2649
M122   C11   C7  2613
M123   C11   C9  2417
M124   C11   C10  2907
M125   C11   C12  2585
M126   C11   C14  2698
M127   C11   C15  2509
M128   C12   C2  2949
M129   C12   C6  2320
M130   C12   C8  2741
M131   C12   C11  2754
M132   C12   C14  2606
M133   C13   C0  2260
M134   C13   C1  2877
M135   C13   C3  2199
M136   C13   C5  2628
M137   C13   C7  2227
M138   C13   C11  2369
M139   C13   C12  2829
M140   C13   C14  2490
M141   C14   C0  2237
M142   C14   C1  2931
M143   C14   C2  2328
M144   C14   C3  2033
M145   C14   C4  2391
M146   C14   C5  2623
M147   C14   C7  2995
M148   C14   C8  2302
M149   C14   C9  2529
M150   C14   C10  2099
M151   C14   C12  2757
M152   C14   C13  2562
M153   C14   C15  2752
M154   C15   C0  2695
M155   C15   C3  2587
M156   C15   C5  2824
M157   C15   C6  2101
M158   C15   C7  2653
M159   C15   C8  2421
M160   C15   C9  2542
M161   C15   C10  2666
M162   C15   C12  2875
M163   C15   C13  2855
M164   C15   C14  2934
