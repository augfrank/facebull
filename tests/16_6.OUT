35038
0 16 21 36 45 63 66 75 93 99 114 119 132 137 150 157
